utils/src/bitwise_operations.rs

```diff
@@ -56,6 +88,17 @@ fn pad(input: &BigUint, bytes: usize) -> Vec<u8> {
     padded
 }
 
+// Transforms a vector of bits in little endian to a BigUint
+fn le_bitvec_to_biguint(input: &[bool]) -> BigUint {
```

Comment:
A better way is to implement [`ToBigUint`](https://docs.rs/num-bigint/0.1.35/num_bigint/trait.ToBigUint.html) on `[bool; N]`, `&[bool]`, or `Vec<bool>`

Reply:
But one cannot implement a trait defined somewhere else for native types

Reply:
I will redefine ToBigUint then